spec/features/browsing_authors_spec.rb

```diff
@@ -23,11 +26,24 @@
         end
 
         scenario "I should see the author's detailed information" do
-          within '.author .name' do
+          within '.author-name' do
             expect(page).to have_content authors.first.name
           end
         end
-        scenario 'I can see the zines created by a given author'
+
+        scenario 'I can see the zine titles created by a given author' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.